po/fa.po

```diff
@@ -491,45 +491,44 @@ msgstr "اشاره‌گرهای هوشمند"
 #: src/SUMMARY.md src/smart-pointers/box.md:1
 #: src/android/interoperability/cpp/type-mapping.md:9
 msgid "`Box<T>`"
-msgstr ""
+msgstr "`<Box<T`"
 
 #: src/SUMMARY.md src/smart-pointers/rc.md:1
 msgid "`Rc`"
-msgstr ""
+msgstr "`Rc`"
 
 #: src/SUMMARY.md src/smart-pointers.md src/smart-pointers/trait-objects.md:1
-#, fuzzy
 msgid "Owned Trait Objects"
-msgstr "آبجکت‌های موصوفی (Trait Objects)"
+msgstr "Owned Trait Objects"
 
 #: src/SUMMARY.md src/smart-pointers.md src/smart-pointers/exercise.md:1
 msgid "Exercise: Binary Tree"
-msgstr ""
+msgstr "تمرین: درخت دودویی"
 
 #: src/SUMMARY.md
 msgid "Day 3: Afternoon"
-msgstr "روز سوم: بعد از ظهر"
+msgstr "روز ۳: بعد از ظهر"
 
 #: src/SUMMARY.md src/running-the-course/course-structure.md:61
 #: src/welcome-day-3-afternoon.md src/borrowing.md
 msgid "Borrowing"
-msgstr "قرض گرفتن"
+msgstr "قرض‌گیری"
```

Comment:
In the RUST , the word **Borrowing** is a concept.
If you are translating, put the English equivalent in parentheses next to the translated text.